website/src/errorsIdentifiers.json

```diff
@@ -9732,7 +9732,7 @@
             ]
         }
     },
-    "reeturn.unionTypeNotSupported": {
+    "return.unionTypeNotSupported": {
```

Comment:
This needs to be fixed in phpstan-src.

Reply:
Done in https://github.com/phpstan/phpstan-src/pull/3176
reverting here ...